xsnippet/api/default.conf

```diff
@@ -35,7 +35,102 @@ connection = mongodb://localhost:27017/xsnippet
 ;        python
 ;        javascript
 ;
-; By default, constraint is not defined so any syntax is accepted.
+
+syntaxes =
```

Comment:
Do you have strong on opinion on filling `default.conf` with syntaxes? I'm just thinking that having it empty would be better for default as some might want to do not restrict submitted snippets to this set of syntaxes. Even more, we might want to change the highlighting library on front-end side, hence the list of syntaxes will change accordingly.

Reply:
No, I just thought it would make the migration easier. We can perform when it's time.